packages/core/src/lib/pagination/lines.mts

```diff
@@ -0,0 +1,43 @@
+import { rotate, splitLines } from '../utils.mjs';
+import { Paged } from './types.mjs';
+
+type Inputs<T> = {
+  items: readonly T[];
+  width: number;
+  render: (paged: Paged<T>) => string;
+  active: number;
+  position: number;
```

Comment:
What is the difference between those 2 parameters?

Reply:
`active` is the item in the list we need to point at; `position` is the position at which we expect the active item to be rendered. for example, for single-line renders of alphabets, active 2 position 3 is:

```
  Z
  A
  B
> C
  D
```

and active 2 position 1 is:

```
  B
> C
  D
  E
  F
```

for page size of 5.

Reply:
I can add documentation to the parameters on the type, if that works.